### PR TITLE
[SPARK-28591][SPARK-Core] correcting the log statement

### PR DESCRIPTION
**Pull request by sandeep-katta**
## What changes were proposed in this pull request?

Correcting the log statement, `ShuffleBlockFetcherIterator` sends the request to the fetch the blogs.So log statement is incorrect

## How was this patch tested?

Manually

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala [hunk 1]
```diff
@@ -218,7 +218,7 @@ final class ShuffleBlockFetcherIterator(
   }
 
   private[this] def sendRequest(req: FetchRequest) {
-    logDebug("Sending request for %d blocks (%s) from %s".format(
+    logDebug("Sending request for %d blocks (%s) to %s".format(
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala]: I think the statement is correct in that it's saying that it's requesting bytes _from_ the remote address. The request is sent _to_ that address, sure. "from" relates to "blocks" not "Sending".

Comment (sandeep-katta) [reply core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala]: either way it's confusing, shall we change to 

Requesting for %d blocks (%s) from %s"

Comment (srowen) [reply core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala]: (remove 'for')
I don't think that's materially different. I'd just leave it.

Comment (HyukjinKwon) [reply core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala]: Yea, let's don't fix. I don't think it virtually matters.

**Pull request closed without merging**